challenge.js

```diff
@@ -42,7 +43,7 @@
  */
 
 // ANSWER
-
+// document.querySelector("p")
```

Comment:
We were asking for a selector rather than JS. But the selector you gave was right - it will refer to that `<p>` element (along with any others that might exist).